instructor_notes/11_git-init-server.md

```diff
@@ -0,0 +1,171 @@
+  * Once the token is created, copy/paste it and save in a (local) _secure location_.
+  Emphasize that if they have a password manager, they should use it!
+* In the R Console in the RStudio Server, use the `gitcreds` package to store this information:
+**Before pasting the token, make sure screen share is off.**
+  * Run `gitcreds::gitcreds_set()` and paste the PAT into Console.
+  * Clear the Console so the PAT is no longer visible before resuming screen share.
+* Run `gitcreds::gitcreds_get()` to ensure a PAT has been set.
+They should see this output if it was successful:
+
+    ```r
+    > gitcreds::gitcreds_get()
+    <gitcreds>
+    protocol: https
+    host    : github.com
+    username: PersonalAccessToken
+    password: <-- hidden -->
+    ```
+* Explain that they will have to repeat this step every 12 hours to be able to continue communicating with GitHub because of how we set up caching on RStudio Server.
+
+## Part 2: Create a git repository
+
+* In the Terminal, navigate to the `training-modules` workshop directory and create a repository:
+
+```sh
+# Navigate to the training-modules directory
+cd ~/training-modules
+
+# Create a git repository
+git init
+```
+
+* Before adding and committing, take a moment to look at the `.gitignore` file present in this directory to see what we expect to be added vs ignored.
+
+* Run the following code to add, commit, and push the new repo to GitHub:
+```sh
+# Add all (non-ignored) files to the directory
+# Explain that we generally like to add files one at a time, but this strategy is appropriate for
+#  the context of initializing a repo, especially given the presence of the .gitignore file!
+git add .
+
+# Throw in a git status for good measure (run as often as desired while using git!)
+git status
+
+# Make a commit
+git commit -m "First commit for RNA-seq workshop materials repo"
+```
+
+* Next, we need a corresponding remote repository to push to.
+For this, instruct participants to create a new repo on GitHub.com
+    * _Do not_ initialize the repository with any files (no `README`, `.gitignore`, or `LICENSE`)
+
+* Copy/paste the `https` URL (`{REMOTE-URL}`) and use for the following code back in R:
+```sh
+# Set the remote repo address
+git remote add origin {REMOTE-URL}
+
+# push to main
+# The `-u origin main` part is only needed the first time you push to a branch that you have not pushed to before,
+#  or does not exist on the remote
+# Moving forward in this branch, pushing will go to this URL by default
+git push -u origin main
+```
+
+## Next steps
+
+Explain that after we complete each training notebook, they should run the following to sync their local repo with GitHub:
+
+```sh
+# add files one at a time
+git add <specific .Rmd file>
+git add <specific .html file>
+
+git commit -m "Informative message about notebook being added"
+
+git push
+```
+
+Note that if they wanted to create a new branch and push it to GitHub, they would do:
+
+```sh
+# create new branch
```

Comment:
It is probably worth noting that they do not need `-c` for general switching among branches; adding a little instruction about switching back to `main` or to a different branch?

```suggestion
# create new branch and switch to it
```